### PR TITLE
Update windows cmd line 101 to terminal

### PR DESCRIPTION
**Pull request by gtrufitt**
_Note:_ This isn't finished but I wanted to get some initial feedback and see if anyone thinks anything needs adding.

I need to run through this on a non customised machine - having zsh and things on my PATH already makes it difficult to be certain that it will work as expected on the edu centre machines. I can run bash but still, I don't think it's as it would be on the other machines.

### CODE REVIEW

File: source/appendix_mac.rst [hunk 1]
```diff
@@ -18,15 +18,14 @@ Terminal 101
...
-The `cmd.exe` program offers the same interaction but using typed commands:
+The `terminal` program offers the same interaction but using typed commands:
     
-When `cmd.exe` launches you get a prompt:: 
+When `terminal` launches you get a prompt:: 
 
-    C:\Users\greg-lo>
+    ~ >
 
 The prompt gives you your currnent location followed by a `>`.
```
Comment (susiecoleman) [inline source/appendix_mac.rst]: current typo

File: source/appendix_mac.rst [hunk 2]
```diff
@@ -18,15 +18,14 @@ Terminal 101
...
 
-    C:\Users\greg-lo>
+    ~ >
 
 The prompt gives you your currnent location followed by a `>`. 
-Here I am in the directory `greg-lo` which itself is in the directory `Users`. 
-`C` refers the hardrive I am on.
+Here I am in the directory `COMPLETE THIS` which itself is in the directory `Users`. 
 
 Now you enter the `dir` command::
```
Comment (susiecoleman) [inline source/appendix_mac.rst]: dir command is windows

**Comment by susiecoleman** [comment]
Few minor comments, I think it would be good to see what it's like running it on the education centre. I don't think you need to add any more. Maybe how to use `man` to find out more about commands but that's definitely outside the scope of the course. Also if you add `man` we should definitely tell them that `q` is how you quit the man pages.

👍

**Comment by gtrufitt** [comment]
@maxspencer Lets review this again today with the notes you took :)

**Comment by gtrufitt** [comment]
I've updated the debugging appendix in this PR too

**Pull request merged**